x/gamm/pool-models/balancer/pool.go

```diff
@@ -625,10 +625,10 @@ func (p Pool) SpotPrice(ctx sdk.Context, baseAsset, quoteAsset string) (spotPric
 		return sdk.Dec{}, errors.New("pool is misconfigured, got 0 weight")
 	}
 
-	// spot_price = (Base_supply / Weight_base) / (Quote_supply / Weight_quote)
-	// spot_price = (weight_quote / weight_base) * (base_supply / quote_supply)
-	invWeightRatio := quote.Weight.ToDec().Quo(base.Weight.ToDec())
-	supplyRatio := base.Token.Amount.ToDec().Quo(quote.Token.Amount.ToDec())
+	// spot_price = (Quote_supply / Weight_quote) / (base_supply / Weight_base)
```

Comment:
Note to reviewers: Please double check if this equation is correct

Reply:
This LGTM, thoughts on adding a simple example in the code comment?